docs/configuration.md

```diff
@@ -0,0 +1,139 @@
+# Configuring and Tuning Cluster Logging
+
+OKD cluster logging is configurable by modifying the CustomResource deployed
+in the **openshift-logging** namespace.  The [sample](../hack/cr.yaml) resource
+defines a complete cluster logging deployment that includes all the subcomponents
+of the logging stack to collect, store and visualize logs.  The `cluster-logging-operator`
+watches the a `ClusterLogging` CustomResources and adjusts the logging deployment accordingly.
+
+## Management State
+Cluster logging is managed by the `cluster-logging-operator`.  It can be placed
+into an unmanaged state allowing an administrator to assume full control of individual
+component configurations and upgrades by changing the `managementState` from `Managed` to `Unmanaged`.
+**Note:** An unmanaged deployment will not receive updates until the `ClusterLogging` custom resource is placed back into a managed state.
+```
+  spec:
+    managementState: "Managed"
+```
+
+## Component Images
+The image for each subcomponent are overwriteable by modifying the appropriate
+environment variable in the `cluster-logging-operator` Deployment.  Each variable
+takes a full pull spec of the image:
+
+* `CURATOR_IMAGE`
+* `ELASTICSEARCH_IMAGE`
+* `FLUENTD_IMAGE`
+* `KIBANA_IMAGE`
+* `OAUTH_PROXY_IMAGE`
+* `RSYSLOG_IMAGE`
+
+## Common Configurations
+### Memory and CPU
+Each component specification allows for adjustments to both the CPU and 
+memory limits.  This is defined by modifying the `resources`
+block with valid memory (e.g. 16Gi) and CPU values (e.g 1):
+```
+  spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        resources:
+          limits:
+            cpu:
+            memory:
+          requests:
+            cpu:
+            memory:
+```
+
+## Data Aggregation and Storage
+An Elasticsearch cluster is responsible for log aggregation.  Following is a sample 
+of the spec:
+```
+  spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        nodeCount: 3
```

Comment:
we may want to note that there is the ability to specify `resources` for the cluster here (will be the same for each node)

Reply:
Added text to `Common Configuration`